packages/react-components/src/types/VideoGalleryParticipant.ts

```diff
@@ -90,4 +90,10 @@ export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
   isSpeaking?: boolean;
   /** Video stream of shared screen */
   screenShareStream?: VideoGalleryStream;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** Call state that the participant is in.
+   *
+   * If nothing is provided here the participant will appear as if in Connected state.
+   */
+  state?: 'Connecting' | 'Connected' | 'Ringing' | 'Hold' | 'Idle' | 'InLobby';
```

Comment:
is videogallery using all these different participant states? If not scope down to only what is being used by the video gallery